lms/templates/seq_module.html

```diff
@@ -19,8 +19,8 @@
            data-page-title="${item['page_title']|h}"
            aria-controls="seq_contents_${idx}"
            id="tab_${idx}"
-           tabindex="0"
-           role="tab">
+           tabindex="0">
+            <i class="icon fa seq_${item['type']}"></i>
```

Comment:
Good news, `${item['type']}` is available as a string. I've got it included as `sr` text before the section name. So it'll say "Video: [name of section]", or "Problem: [name of section]".